src/app/utils/state.ts

```diff
@@ -126,7 +144,8 @@ class StateManager {
     if (error) {
       console.error("error: ", error);
     }
-    return data![0].status;
+
+    return data![0];
```

Comment:
**issue:** Add null check before accessing data[0] to prevent runtime errors

The current code assumes data is non-null and has at least one element. This could crash if the query returns no results. Consider adding appropriate null checks and error handling.